client/my-sites/site-settings/site-visibility/launch-site.jsx

```diff
@@ -116,4 +116,10 @@ const LaunchSite = () => {
 	);
 };
 
-export default LaunchSite;
+const mapDispatchToProps = ( dispatch ) => {
+	return {
+		dispatchLaunchSite: ( siteID ) => dispatch( launchSite( siteID ) ),
+	};
+};
+
+export default connect( null, mapDispatchToProps )( LaunchSite );
```

Comment:
Could you use the `useDispatch` hook instead of connect/react-redux? 

Looks like that was the original bug. (Nice work figuring that out btw!)

```
import { useDispatch } from '@wordpress/data';
...
const dispatch = useDispatch();
```

Reply:
That's great, I always forgot about `useDispatch` 🤦‍♂️ 
I'll update the PR

Reply:
That makes at least three of us :laughing: